test_chan/README.md

```diff
@@ -13,9 +13,17 @@ $ virtualenv .
 $ bin/python setup.py install
```

Comment:
Looks good but you shouldn't need 2 READMEs -- I think github renders .rst if you want to stick w/ a python docutils based set of docs here

Reply:
github definitely handles README.rst e.g. https://github.com/Pylons/pyramid